admin_guide/sdn_troubleshooting.adoc

```diff
@@ -495,6 +495,54 @@ normal operation, so do not be surprised to see entries there.
 
 Check external firewalls, if any, allow traffic to the target address (this is site-dependent, and beyond the purview of this guide).
 
+== Debugging Virtual Networking
+
+=== Builds on a Virtual Network are Failing
+
+If you are installing OpenShift using a virtual network (for example,
+OpenStack), and a build is failing, the node maximum transmission unit (MTU)
+might not be compatible with the network MTU (for example, eth0).
```

Comment:
Reading further, i see that "network MTU" is a shorthand for "MTU of the primary network interface".  I suppose if everyone knows that, then there is no need to change the example (please disregard previous line-comment).

Still, might be a good idea to mark up eth0, like `eth0`.